frontend/ui/src/reducers/data/conversations/index.ts

```diff
@@ -209,6 +209,10 @@ function allReducer(
               // Ensure that there is always a display name present
               ...pick(state.items[action.payload.identifier]?.metadata, 'contact.displayName'),
               ...(<MetadataEvent<ConversationMetadata>>action.payload).metadata,
+              contact: {
```

Comment:
The `pick ` line above its meant to fix this very issue 🤔 So having both is probably not good.